buildlibxml.py

```diff
@@ -38,9 +38,14 @@ def download_and_extract_windows_binaries(destdir):
         if release_path in filename
     ]
 
-    arch = "win64" if sys.maxsize > 2**32 else "win32"
     if sys.version_info < (3, 5):
         arch = 'vs2008.' + arch
+    elif platform.machine() == 'ARM64':
+        arch = "win-arm64"
+    elif sys.maxsize > 2**32:
+        arch = "win64"
+    else:
+        arch = "win32"
```

Comment:
This diff breaks the `arch` reference in the first if-block and breaks the py27 builds on appveyor:
- https://ci.appveyor.com/project/scoder/lxml/builds/41249423/job/vnu6gmaa3u0f4by7#L49
- https://ci.appveyor.com/project/scoder/lxml/builds/41249423/job/173d08dq3bbx6mef#L57

Not sure how relevant this is considering you're currently trying to sort out the dependency builds in the remote repository, but just wanted to let you know in case you haven't noticed already.

Reply:
Yes I think the statements in lines 41 and 42 should be moved after line 48

Reply:
Opened https://github.com/lxml/lxml/pull/331 to fix it

Reply:
Thanks, yeah, I noticed this, too, when I was trying to set up an automated wheel build.

Reply:
(Sorry for breaking your original PR)